src/main.cpp

```diff
@@ -107,8 +112,30 @@ bool download(DownloadEnum::Category cat, const char* name)
 	return true;
 }
 
+#ifdef _WIN32
+
+static std::vector<std::string> argv_strings;
+static std::vector<const char*> argv_data;
```

Comment:
Why are these two defined globally? They are only used in the `prepare_args` function, so I would just define them there (as static obviously, so they don't get deallocated, causing segmentation fault.)

Reply:
No reason :), draft code :)